packages/ember-test-app/app/templates/index.hbs

```diff
@@ -51,4 +51,14 @@
   </div>
 </div>
 
-{{outlet}}
\ No newline at end of file
+{{outlet}}
+
+<Footer>
+  <:left>
+    <a href="">&copy; Knoxville Utility Board | KUB.org</a>
+  </:left>
```

Comment:
Talked with Lucas during PR review go again and remove this from the dummy app for now. We aren't sure that will go in the final design
```suggestion
```